Client/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -1021,13 +1039,32 @@ std::variant<bool, CLuaMultiReturn<float, float, float>> CLuaPedDefs::GetElement
 
 std::variant<bool, CLuaMultiReturn<float, float, float>> CLuaPedDefs::GetElementBoneRotation(lua_State* const luaVM, CClientPed* entity, std::int32_t boneId)
 {
+    if (boneId > BONE_RIGHTFOOT)
```

Comment:
This check is incomplete. Because you dediced to use signed int. Please, replace in with uint or check whether a number is higher than zero or not.

Reply:
as you say, fixed